packages/web-components/src/components/ic-checkbox/ic-checkbox.tsx

```diff
@@ -127,6 +128,9 @@ export class Checkbox {
 
   componentWillLoad(): void {
     addFormResetListener(this.host, this.handleFormReset);
+    this.host
+      .querySelector("ic-text-field")
+      ?.addEventListener("icChange", (e) => e.stopImmediatePropagation());
```

Comment:
we should also remove the event listener in the disconnected callback. see similar code in navigation group